### DISCUSSION TITLE
No widget options available for WYSIWYG on DataObjects

### DISCUSSION THREAD

**Discussion by anjali-pajapati** [pimcore/pimcore#15648] [General]
Hi Team
I recently downloaded and setup the new Pimcore Instance(v11.0.5) on local and created some data modelling.

When I created a WYSIWYG field in a class and then created an object, I was not able to see any input widget options regarding the WYSIWYG like Bold, Italic, source etc..
The field is present but not any options.

Is this a bug or I am missing any configuration?

![image](https://github.com/pimcore/pimcore/assets/140892392/44f2b3f8-5773-44bb-80ac-c9c7a6b8ee81)

![image](https://github.com/pimcore/pimcore/assets/140892392/626636c6-036c-49c2-997e-617060463afb)

**_This is not reproducible on Demo website but one of my collegue also tried the same and facing the same issue_**
![image](https://github.com/pimcore/pimcore/assets/140892392/4241be93-740a-48d0-8ebd-fc60aee8e680)

**Comment by blankse** [comment]
@anjali-pajapati Did you activate the PimcoreTinymceBundle?
https://github.com/pimcore/demo/blob/56de4de661aa8294aa62204be1200f611c5e17f0/config/bundles.php#L41

**Reply by anjali-pajapati** [reply]
Thanks @blankse for your input. 
The above link helped but I also had to do an additional change in the DB.

1. **Go to _config/bundles.php_ and add below code**
```
<?php

use Pimcore\Bundle\TinymceBundle\PimcoreTinymceBundle;

return [
    //Twig\Extra\TwigExtraBundle\TwigExtraBundle::class => [ 'all' => true ],
    PimcoreTinymceBundle::class => [ 'all' => true ],
];
```


2. **Add this value in DB**
`INSERT INTO settings_store (id,scope,data,type) VALUES ('BUNDLE_INSTALLED__Pimcore\\Bundle\\TinymceBundle\\PimcoreTinymceBundle', 'pimcore', '1','bool');`

Now, the widget options are available correctly.

**Reply by blankse** [reply]
@anjali-pajapati You should use the bundle install command to install the bundles. Some bundle create also permissions and own db tables:
`bin/console pimcore:bundle:install PimcoreTinymceBundle`
